default/generated/eap6/63-websphere-xml.windup.yaml

```diff
@@ -68,8 +68,7 @@
           webappext: webappext.xmi
         xpath: /webappext:WebAppExtension
       from: files
-    - as: default
```

Comment:
@milderhc , Upstream has entirely removed this file in the following commit: https://github.com/konveyor/rulesets/commit/d930baf2645d6a7ef9486c4f81bc40d8abc3a6af#diff-c2e5e982401374cada0d7ab57243649bc0bff3e30ff82b6e2f75504a259e8092. We can proceed with removing it downstream as well, as the eap6 rules are integrated by upstream. There are no downstream specific changes for eap6 folder. I will create a task to cherry-pick the commit from upstream.